libexec/ipmi

```diff
@@ -78,10 +78,10 @@ usage()
 {
   echo "(c) 2008-2011 Alces Software Ltd & Stephen F Norledge"
   echo "alces-ipmi command, perform ipmi commands on single or multiple machines"
-  echo "usage: alces ipmi [-g] <name> -c <command>"
```

Comment:
Do we need any of this `usage` now - won't any  issue with using the `ipmi` command be handled by the Metalware CLI and so this will never be displayed? If so can we remove it?